src/openrct2/scripting/ScPark.hpp

```diff
@@ -24,6 +25,209 @@
 
 namespace OpenRCT2::Scripting
 {
+    static constexpr const char* ParkMessageTypeStrings[] = { "attraction", "peep_on_attraction", "peep",   "money",
+                                                              "blank",      "research",           "guests", "award",
+                                                              "chart" };
```

Comment:
This is so weird, I'd rather break after the `{` and have one per line than this. :X

Reply:
This is clang-format, not me.

Reply:
Add a comma to the end, and clang-format will put them all one one line.